.circleci/config.yml

```diff
@@ -1,23 +1,27 @@
-# Python CircleCI 2.0 configuration file
-#
-# Check https://circleci.com/docs/2.0/language-python/ for more details
-#
 version: 2
 jobs:
   test:
     docker:
-      - image: cimg/go:1.14.1
+      - image: cimg/go:1.19
+        environment:
+          GOFLAGS: -p=8
     steps:
       - checkout
       - run:
-          name: reinstall certs package
-          command: sudo apt-get update && sudo apt-get install --reinstall ca-certificates  
+          name: Avoid hosts unknown for github
+          command: echo -e "Host github.com\n\tStrictHostKeyChecking no\n" >> ~/.ssh/config
```

Comment:
Shady stuff

Reply:
Maybe but we are using this everywhere ;)